pkg/common/config/config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"io/ioutil"
+	log "github.com/sirupsen/logrus"
+)
+
+var initContainerTemplate string
+
+func init() {
+	bytes, err := ioutil.ReadFile("/etc/config/initContainer.yaml")
+	if err != nil {
+		log.Warningf("error while read initContainerTemplate, use default. error: %s", err)
+		setDefaultInitContainerTemplate()
+	} else {
+		initContainerTemplate = string(bytes)
+	}
+}
+
+func setDefaultInitContainerTemplate() {
+	initContainerTemplate = `
+- name: init-pytorch
+  image: busybox
+  command: ['sh', '-c', 'until nslookup {{.MasterAddr}}; do echo waiting for master; sleep 2; done;']`
+}
```

Comment:
Define the default init container template as a const string will be better, no need to add a function here.